distribution/src/conf/wso2server.sh

```diff
@@ -237,10 +237,13 @@ do
         CARBON_CLASSPATH="$CARBON_CLASSPATH":$f
     fi
 done
-for t in "$CARBON_HOME"/../lib/commons-lang*.jar
+for t in "$CARBON_HOME"/wso2/lib/commons-lang*.jar
 do
     CARBON_CLASSPATH="$CARBON_CLASSPATH":$t
 done
+
+
+
```

Comment:
Do we need all these extra lines?

Reply:
Fixed with : https://github.com/wso2/micro-integrator/pull/90